packages/ddex/Dockerfile

```diff
@@ -18,32 +24,42 @@ RUN npm install turbo --global
 COPY . .
 RUN turbo prune --scope=@audius/ddex-client --scope=@audius/ddex-server --docker
 
-# Install and build client and server dists and run app
-FROM base AS runner
+# Second stage: Install and build client and server dists
+FROM base AS app-builder
 
 WORKDIR /app
-
 RUN apk add --no-cache python3 py3-pip make g++ curl bash libc6-compat
 RUN apk update
 
 # First install dependencies (as they change less often)
 COPY .gitignore .gitignore
-COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/package-lock.json ./package-lock.json
-COPY --from=builder /app/scripts ./scripts
+COPY --from=turbo-builder /app/out/json/ .
+COPY --from=turbo-builder /app/out/package-lock.json ./package-lock.json
+COPY --from=turbo-builder /app/scripts ./scripts
 
 RUN CI=true npm i
 
 # Build the app and its dependencies
-COPY --from=builder /app/out/full/ .
+COPY --from=turbo-builder /app/out/full/ .
 COPY svgr-template.js svgr-template.js
 COPY turbo.json turbo.json
 RUN npx turbo run build --filter=@audius/ddex-client --filter=@audius/ddex-server
 
-CMD ["sh"]
-# COPY packages/ddex/client/dist /app/packages/ddex/server/public
+# Make a smaller image by removing all src directories (except for in node_modules)
+RUN mv packages/ddex/client/dist packages/ddex/server/public
+RUN rm -rf packages/ddex/client
+RUN find packages -path '*/node_modules/*' -prune -o -name 'src' -type d -exec rm -rf {} +
+
+# Final stage: Create a runnable image
```

Comment:
why separate the runner from the app-builder and not just 
```
WORKDIR /app/packages/ddex/server

EXPOSE 8926
CMD ["node", "dist/index.js"]
```
from here?

Reply:
i was trying to do the standard approach where you copy only what's needed, but it's difficult with hoisted node_modules and locally built deps. i guess combined or not it's pretty similar